pkg/controller/replicaset/replica_set_test.go

```diff
@@ -87,7 +89,7 @@ func skipListerFunc(verb string, url url.URL) bool {
 	if verb != "GET" {
 		return false
 	}
-	if strings.HasSuffix(url.Path, "/pods") || strings.HasSuffix(url.Path, "/replicasets") {
+	if strings.HasSuffix(url.Path, "/pods") || strings.Contains(url.Path, "/replicasets") {
```

Comment:
The fakeHandlers that use this skipListerFunc are really just looking for updates (PUT). I hit a case where due to the changes, an additional GET happened on a subresource of `/replicasets`, so I expanded the set of paths that get ignored.